config/locales/helpers.en.yml

```
@@ -36,6 +37,7 @@ en:
         age_range_min: From
         age_range_max: To
         age_range_note: If you've entered a new range please explain why (optional)
+        failure_reason_notes: "Note to the applicant"
```

```suggestion
        failure_reason_notes: Note to the applicant
```

The quotes can mostly be avoided when using YAML.